messaging/src/main/java/org/axonframework/commandhandling/gateway/AbstractRetryScheduler.java

```diff
@@ -136,6 +177,8 @@ public boolean scheduleRetry(CommandMessage commandMessage,
 
```

Comment:
The class JavaDoc should also state the defaults set. So, in this case, the defaulted failures list should be part of it.

Furthermore, it should also be part of the `builder()` method JavaDoc of the concrete implementation of the `AbstractRetryScheduler`.
That way, users can quickly see what's defaulted and not right when to construct their main builder.